uint8 setpointType
float64 x
float64 y
float64 z
float64 yaw
